### ISSUE TITLE
Domain name representations: freshports.org 

### ISSUE THREAD

**Issue by grahamperrin** [FreshPorts/freshports#412]
For consistency, and beautification, please consider changing _FreshPorts.org_ (representations) to lowercase:

* _freshports.org_

Inconsistencies became noticeable when I looked, in detail (before aiming to redact something), at a daily notification email: 

* mixed upper and lowercase `From …@FreshPorts.org`
* mixed upper and lowercase in three places in the content of the email
* lowercase in all other parts of the content.

Lowercase is true to what's seen in reality. Emphasised here by Firefox: 

![image](https://user-images.githubusercontent.com/192271/208612053-88dceaba-7d26-4490-815c-551ef8444ec9.png)

----

Note, this plea relates to _freshports.org_ – the domain – not more broadly to _FreshPorts_. 

The latter, not signifying a domain, is perfect.

Thanks

**Comment by dlangille** [comment]
In summary, this is about URLs. 

Ya know, an example email would be useful here.

```
Port count: 57881 https://www.freshports.org/categories.php
 New ports:    12 https://www.freshports.org/ports-new.php?interval=today

24 Dec 2022

https://www.freshports.org/audio/tuner/
     Minimalist radio station player

https://www.freshports.org/science/bout++/
     Plasma fluid finite-difference simulation code
```

I found:

```
[dev-ingress01 dan ~/scripts] % grep -r 'Port count' *                                                                                                                   18:54:12
hourly_stats.pl:		print FILE '<TR><TD><A HREF="/categories.php" TITLE="Number of ports in the database">Port count</A></TD> <TD>'      . $Stats{new}       . '</TD></TR>' . "\n";
report-maintainer-ports.pl:	$BodyHeader .= "Port count: " . sprintf("%5u", $PortCount) . " https://www.FreshPorts.org/categories.php\n";
report-new-ports.pl:	$BodyHeader .= "Port count: " . sprintf("%5u", $PortCount) . " https://www.FreshPorts.org/categories.php\n";
report-notification.pl:	$BodyHeader .= "Port count: " . sprintf("%5u", $PortCount) . " https://www.FreshPorts.org/categories.php\n";
report-security-notice.pl:	$BodyHeader .= "Port count: " . sprintf("%5u", $PortCount) . " https://www.FreshPorts.org/categories.php\n";
```

It might be better to use the constants provided elsewhere.

**Comment by dlangille** [comment]
No such constant exists. That's for another day.

```
[dev-ingress01 dan ~/scripts] % svn di
Index: announce.pl
===================================================================
--- announce.pl	(revision 5837)
+++ announce.pl	(working copy)
@@ -51,9 +51,9 @@
 --
 
 You are receiving this message as part of the service
-you joined at https://www.FreshPorts.org/ but if you no longer
+you joined at https://www.freshports.org/ but if you no longer
 wish to receive such messages, please go to
-https://www.FreshPorts.org/report-subscriptions.php
+https://www.freshports.org/report-subscriptions.php
 
 If a problem occurs, please send details, including the email
 address in question, to postmaster\@freshports.org
Index: report-maintainer-ports.pl
===================================================================
--- report-maintainer-ports.pl	(revision 5837)
+++ report-maintainer-ports.pl	(working copy)
@@ -161,8 +161,8 @@
 	
 	my $BodyHeader = '';
 	$BodyHeader .= $Announce . "\n"; 
-	$BodyHeader .= "Port count: " . sprintf("%5u", $PortCount) . " https://www.FreshPorts.org/categories.php\n";
-	$BodyHeader .= " New ports: " . sprintf("%5u", $NewPorts)  . " https://www.FreshPorts.org/ports-new.php?interval=$Interval\n\n";
+	$BodyHeader .= "Port count: " . sprintf("%5u", $PortCount) . " https://www.freshports.org/categories.php\n";
+	$BodyHeader .= " New ports: " . sprintf("%5u", $NewPorts)  . " https://www.freshports.org/ports-new.php?interval=$Interval\n\n";
 
 	my $wrapper = Text::Wrapper->new(columns => 72, body_start => '  ');
 
Index: report-new-ports.pl
===================================================================
--- report-new-ports.pl	(revision 5837)
+++ report-new-ports.pl	(working copy)
@@ -133,8 +133,8 @@
 
 	my $BodyHeader = '';
 	$BodyHeader .= $Announce . "\n"; 
-	$BodyHeader .= "Port count: " . sprintf("%5u", $PortCount) . " https://www.FreshPorts.org/categories.php\n";
-	$BodyHeader .= " New ports: " . sprintf("%5u", $NewPorts)  . " https://www.FreshPorts.org/ports-new.php?interval=$Interval\n\n";
+	$BodyHeader .= "Port count: " . sprintf("%5u", $PortCount) . " https://www.freshports.org/categories.php\n";
+	$BodyHeader .= " New ports: " . sprintf("%5u", $NewPorts)  . " https://www.freshports.org/ports-new.php?interval=$Interval\n\n";
 
 	$Body = $BodyHeader;
 	while ($row = $sth->fetchrow_hashref()) {
Index: report-notification.pl
===================================================================
--- report-notification.pl	(revision 5837)
+++ report-notification.pl	(working copy)
@@ -158,8 +158,8 @@
 	
 	my $BodyHeader = '';
 	$BodyHeader .= $Announce . "\n"; 
-	$BodyHeader .= "Port count: " . sprintf("%5u", $PortCount) . " https://www.FreshPorts.org/categories.php\n";
-	$BodyHeader .= " New ports: " . sprintf("%5u", $NewPorts)  . " https://www.FreshPorts.org/ports-new.php?interval=$Interval\n\n";
+	$BodyHeader .= "Port count: " . sprintf("%5u", $PortCount) . " https://www.freshports.org/categories.php\n";
+	$BodyHeader .= " New ports: " . sprintf("%5u", $NewPorts)  . " https://www.freshports.org/ports-new.php?interval=$Interval\n\n";
 
 	my $wrapper = Text::Wrapper->new(columns => 72, body_start => '  ');
 
Index: report-security-notice.pl
===================================================================
--- report-security-notice.pl	(revision 5837)
+++ report-security-notice.pl	(working copy)
@@ -162,8 +162,8 @@
 	
 	my $BodyHeader = '';
 	$BodyHeader .= $Announce . "\n"; 
-	$BodyHeader .= "Port count: " . sprintf("%5u", $PortCount) . " https://www.FreshPorts.org/categories.php\n";
-	$BodyHeader .= " New ports: " . sprintf("%5u", $NewPorts)  . " https://www.FreshPorts.org/ports-new.php?interval=$Interval\n\n";
+	$BodyHeader .= "Port count: " . sprintf("%5u", $PortCount) . " https://www.freshports.org/categories.php\n";
+	$BodyHeader .= " New ports: " . sprintf("%5u", $NewPorts)  . " https://www.freshports.org/ports-new.php?interval=$Interval\n\n";
 
 	my $wrapper = Text::Wrapper->new(columns => 72, body_start => '  ');
 
[dev-ingress01 dan ~/scripts] % 
```

**Comment by dlangille** [comment]
Committed revision 5887.

It will be a while before this change hits production.

**Comment by grahamperrin** [comment]
Sweet. Thanks.

**Comment by grahamperrin** [comment]
> … Ya know, an example email would be useful here. …

I did have one nearly prepared on the 20th, got (heavily) distracted by <https://bugs.documentfoundation.org/show_bug.cgi?id=152627>, <https://bugs.documentfoundation.org/show_bug.cgi?id=85295#c16>, <https://bugs.documentfoundation.org/show_bug.cgi?id=143095> and the like. 

Here, although I doubt that you need it now: 

[FreshPorts daily notification - main.pdf](https://github.com/FreshPorts/freshports/files/10314363/FreshPorts.daily.notification.-.main.pdf)

Available on request: the manually edited `.eml` from which I produced the PDF.

**Comment by grahamperrin** [comment]
> In summary, this is about URLs.

…

> Committed revision 5887.

Does that cover the _From_ address? 

`FreshPorts Watch Daemon <FreshPorts-Watch@FreshPorts.org>`

Sorry for the disjointed responses.

**Comment by dlangille** [comment]
It did not cover email addresses. I thought keeping it `FreshPorts.org` was similar to `FreeBSD.org` at https://docs.freebsd.org/en/articles/contributors/

**Comment by grahamperrin** [comment]
Thanks. 

Until a few days ago, I couldn't think of any organisation other than FreeBSD that used uppercase in this way. 

For myself, I avoid it in all contexts other than commit messages, and a handful of other areas where there's strictness (e.g. the `.dot` files for the graphs under <https://wiki.freebsd.org/Developers#Group_Memberships>). 

----

It's lowercase in `Return-Path: <FreshPorts-Watch@freshports.org>` and so on: 

![image](https://user-images.githubusercontent.com/192271/209836207-dafa5614-a01f-4208-b747-08c5b1c04944.png)

In a nutshell: 

* I flinch, mentally, every time I see `FreeBSD.org`, because it's an oft-seen peculiarity
* I never noticed `FreshPorts.org` before 20th December, because the _daily notification_ emails are great for **speed**-reading.

(I looked in **detail** on the 20th, the email [above](https://github.com/FreshPorts/freshports/issues/412#issuecomment-1366648740) (PDF), primarily to see how an author's ID was represented. Only then did I notice the uppercase, it's obscure enough to **not** make me flinch `:-)`)
